docs/UserGuide.md

```diff
@@ -610,18 +611,20 @@ RC4HDB saves your data after every command. There is no need to save manually.
 ---
 
 ### File Command Format:
-* All file commands have a `FOLDER_NAME` field, which signifies the name of the **sub** data folder or the [CSV](#glossary-of-terms) file to be targeted depending on the command that is being used.
-* Do not include any file types in the `FOLDER_NAME`.
-* The following symbols are not to be used in `FOLDER_NAME`.
+* All file commands have a `FOLDER_NAME` or `FILE_NAME` field depending on the command that is being used. 
+* `FOLDER_NAME` signifies the name of the **sub** data folder that is being used.
+* `FILE_NAME` refers to the [CSV](#glossary-of-terms) file to be targeted.
+* Do not include any file types in the `FOLDER_NAME` or `FILE_NAME`.
```

Comment:
Should we use the term file extension? Is that too technical for the user?

Reply:
On second thought, it might be too technical. One way we can dumb it down is to state that the file type is anything beyond the dot i.e. .json